recipes/r-castor/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = '1.3.3' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-castor
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: castor_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/castor_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/castor/castor_{{ version }}.tar.gz
+  sha256: 68e78c5738ab8d418abd52d9bd01ca33a4ff516b2b59c2491c0c89578420ed58
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-rcpp >=0.12.10
+    - r-naturalsort
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - r-rcpp >=0.12.10
+    - r-naturalsort
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('castor')"  # [not win]
+    - "\"%R%\" -e \"library('castor')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=castor
+  license: GPL (>= 2)
+  summary: Efficient tree manipulations for large phylogenies, including pruning, rerooting,
+    calculation of most-recent common ancestors, calculating distances from the tree
+    root and calculating pairwise distances. Calculation of phylogenetic signal and
+    mean trait depth (trait conservatism), ancestral state reconstruction and hidden
+    character prediction of discrete characters. Simulating and fitting models of trait
+    evolution, and generating random trees using birth-death models. Dating trees, comparing
+    two trees, and reading/writing trees to a file. Citation
```

Comment:
The summary is cut off; looks good apart from this!

Reply:
Great thanks! I added the rest of the summary (it was cut-off due to a ":" character I think).